webrtc.html

```diff
@@ -8427,113 +8429,61 @@ <h2 id="sec.stats-model">Statistics Model</h2>
-          "RTCRTPStreamStats" class="dictionary-members">
-            <dt><dfn><code>ssrc</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-            <dt><dfn><code>remoteId</code></dfn> of type <span class=
-            "idlMemberType"><a>DOMString</a></span></dt>
-            <dd>
-              <p>The <code>remoteId</code> can be used to look up the
-              corresponding <code><a>RTCStats</a></code> object that represents
-              stats reported by the other peer.</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
-      <div>
-        <pre class="idl">
-        dictionary RTCInboundRTPStreamStats : RTCRTPStreamStats {
-             unsigned long packetsReceived;
-             unsigned long bytesReceived;
-};</pre>
-        <section>
-          <h2>Dictionary <a class="idlType">RTCInboundRTPStreamStats</a>
-          Members</h2>
-          <dl data-link-for="RTCInboundRTPStreamStats" data-dfn-for=
-          "RTCInboundRTPStreamStats" class="dictionary-members">
-            <dt><dfn><code>packetsReceived</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-            <dt><dfn><code>bytesReceived</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
-      <div>
-        <pre class="idl">
-        dictionary RTCOutboundRTPStreamStats : RTCRTPStreamStats {
-             unsigned long packetsSent;
-             unsigned long bytesSent;
-};</pre>
-        <section>
-          <h2>Dictionary <a class="idlType">RTCOutboundRTPStreamStats</a>
-          Members</h2>
-          <dl data-link-for="RTCOutboundRTPStreamStats" data-dfn-for=
-          "RTCOutboundRTPStreamStats" class="dictionary-members">
-            <dt><dfn><code>packetsSent</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-            <dt><dfn><code>bytesSent</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
+      <h3>Mandatory To Implement Stats</h3>
+      <p>The stats listed in [[WEBRTC-STATS]] are intended to cover a wide
+      range of use cases. Not all of them have to be implemented by every
+      WebRTC implementation.</p>
+      <p>An implementation MUST support generating statistics of the following
+      types when the corresponding objects exist on a PeerConnection, with the
+      attributes that are listed when they are valid for that object:</p>
+      <ul>
```

Comment:
Alpabetizing makes sense. I did that in my implementations except for placing child dictionaries after their parent (e.g. RTC[In/Out]boundRTPStreamStats after RTCRTPStreamStats) since they are related to each other even though that breaks alphabetized order.